src/main/java/com/box/sdk/BoxJSONObject.java

```diff
@@ -73,6 +73,18 @@ public String getPendingChanges() {
     }
 
     /**
+     * Gets a JSON string containing any pending changes to this object that can be sent back to the Box API.
+     * @return a JSON string containing the pending changes.
+     */
+    public JsonObject getPendingChangesAsJsonObject() {
+        JsonObject jsonObject = this.getPendingJSONObject();
```

Comment:
what does this do?

Reply:
added a new method in BoxJSONRequest which can take in JsonObject. This method can be used along with that to set body for a PUT request